src/arkode/arkode.c

```diff
@@ -1300,33 +1305,6 @@ int arkRwtSet(N_Vector y, N_Vector weight, void* data)
   return (flag);
 }
 
-/*---------------------------------------------------------------
-  arkErrHandler is the default error handling function.
-  It sends the error message to the stream pointed to by ark_errfp
-  ---------------------------------------------------------------*/
-void arkErrHandler(int error_code, const char* module, const char* function,
-                   char* msg, void* data)
-{
-  ARKodeMem ark_mem;
-  char err_type[10];
-
-  /* data points to ark_mem here */
-  ark_mem = (ARKodeMem)data;
-
-  if (error_code == ARK_WARNING) { sprintf(err_type, "WARNING"); }
```

Comment:
There are couple uses of `arkProcessError` where the message is intended to be a warning rather than an error.